### ISSUE TITLE
Strange warnings (errors) in output

### ISSUE THREAD

**Issue by dukebarman** [CoatiSoftware/CoatiBugTracker#94]
What are errors in output when start Coati from console?

```
...
    19:08:45 | CxxDeclNameResolver.cpp:280 getDeclName() | ERROR: could not resolve name of decl at: /home/dukebarman/Projects/radare2/shlr/wind/kd.h:63:2
    19:08:45 | CxxDeclNameResolver.cpp:280 getDeclName() | ERROR: could not resolve name of decl at: /home/dukebarman/Projects/radare2/shlr/wind/kd.h:130:2
    30 warnings generated.
    30 warnings generated.
   ...
```

This error (```could not resolve name```) is in all *.h files and some *.c. But Coati is worked normally (maybe :) ).

**Comment by egraether** [comment]
Yes, Coati works correctly. These errors tell us that some names where not resolved correctly. In most of the cases there is not proper name to resolve. We should get rid of these logs.

**Comment by dukebarman** [comment]
I can re-analyse this project and send you a log, if u need of course.

**Comment by egraether** [comment]
No, thank you! :)
We have lots of these cases in our own code that we can look at.

**Comment by dukebarman** [comment]
Ok :) Very wait a fixes!

**Comment by egraether** [comment]
Removed these error logs with 0.6.2.15

**Comment by dukebarman** [comment]
What about this warning?

```
15:12:45 | PersistentStorage.cpp:352 logStats() | WARNING:
```

and this (I think next warns special for projects but why I don't see it in output log with errors / graph / ... or cannot enable view in options?)

```
...
/home/dukebarman/Projects/radare2/libr/anal/p/anal_x86_cs.c:1897:15: warning: unused variable 'in'
                                cs_x86_op in = INSOP(0);
                                          ^
/home/dukebarman/Projects/radare2/libr/anal/p/anal_x86_cs.c:1921:15: warning: unused variable 'in'
                                cs_x86_op in = INSOP(0);
                                          ^
2 warnings generated.
1 warning generated.
...
warning: argument unused during compilation: '-pie'
3 warnings generated.
...
```

**Comment by mlangkabel** [comment]
Hi @dukebarman, I'm sorry for the late reply.

The first Warning is more of an Info. We should really change the type of that log. You can ignore it savely ;)

The other messages appear to be warnings generated by Clang during source code analysis. They do not appear inside the log file because we do not inject them into our logging pipeline. Warnings generated by Clang do not influence the coverage of Coati's indexer at all. So you can ignore them as well. I think it would be best if we suppress those in the future to minimize confusion.

**Comment by dukebarman** [comment]
Hi @mlangkabel, no problem. Thx!

**Comment by mlangkabel** [comment]
Fixed with Coati version 0.7.0.19
